### ISSUE TITLE
Files aren't compressed?

### ISSUE THREAD

**Issue by Peter-Optiway** [cthackers/adm-zip#252]
I created a new zip file that contains a json object.
But the file is not compressed when i compress file with the lowest zip compression in 7zip the file is compressed a lot:
original file: 1 884 KB
Zipped with adm-zip: 1 884 KB
Zipped with 7zip(fastest): 1 134 KB
What is wrong?

```javascript
const AdmZip = require('adm-zip');
var zip = new AdmZip();
var content = JSON.stringify(dataObject);
zip.addFile("legalText.json", Buffer.alloc(content.length, content, 'utf8'));
zip.writeZip(outputPath);
```

**Comment by nathanbussey** [comment]
Any luck resolving this issue?

**Comment by Peter-Optiway** [comment]
Sadly not!

**Comment by vtalas** [comment]
I'm having the same issue here, while I'm trying to compress  the `.js` and `.css`

**Comment by Peter-Optiway** [comment]
This bug seems to be resolved in 0.4.13, good job!
